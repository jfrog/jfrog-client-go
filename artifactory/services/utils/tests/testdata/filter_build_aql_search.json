{
    "results": [
        {
            "repo": "repo1",
            "path": "a",
            "name": "ses.in",
            "type": "file",
            "size": 11,
            "created": "2020-06-25T10:21:17.488+03:00",
            "modified": "2020-06-25T10:21:17.496+03:00",
            "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
            "actual_sha1": "a",
            "properties": [
                {
                    "key": "ca",
                    "value": "1"
                },
                {
                    "key": "build.name",
                    "value": "myBuild"
                }
            ]
        },
        {
            "repo": "repo1",
            "path": "a",
            "name": "no.in",
            "type": "file",
            "size": 11,
            "created": "2020-06-25T10:21:17.502+03:00",
            "modified": "2020-06-25T10:21:17.510+03:00",
            "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
            "actual_sha1": "a",
            "properties": [
                {
                    "key": "c",
                    "value": "1"
                },
                {
                    "key": "build.name",
                    "value": "myBuild"
                },
                {
                    "key": "build.number",
                    "value": "1"
                }
            ]
        },
        {},
        {
            "repo": "repo1",
            "path": "a",
            "name": "no.in",
            "type": "file",
            "size": 11,
            "created": "2020-06-25T10:21:17.502+03:00",
            "modified": "2020-06-25T10:21:17.510+03:00",
            "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
            "actual_sha1": "a",
            "properties": [
                {
                    "key": "c",
                    "value": "1"
                }
            ]
        },
        {},
        {
            "repo": "repo1",
            "path": "a",
            "name": "yes.in",
            "type": "file",
            "size": 11,
            "created": "2020-06-25T10:21:17.502+03:00",
            "modified": "2020-06-25T10:21:17.510+03:00",
            "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
            "actual_sha1": "b",
            "properties": [
                {
                    "key": "css",
                    "value": "1"
                },
                {
                    "key": "build.name",
                    "value": "myBuild"
                }
            ]
        },
        {
            "repo": "repo1",
            "path": "a",
            "name": "no.in",
            "type": "file",
            "size": 11,
            "created": "2020-06-25T10:21:17.502+03:00",
            "modified": "2020-06-25T10:21:17.510+03:00",
            "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
            "actual_sha1": "b",
            "properties": [
                {
                    "key": "css",
                    "value": "1"
                }
            ]
        },
        {
            "repo": "repo1",
            "path": "a",
            "name": "c1.in",
            "type": "file",
            "size": 11,
            "created": "2020-06-25T10:21:17.502+03:00",
            "modified": "2020-06-25T10:21:17.510+03:00",
            "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
            "actual_sha1": "c",
            "properties": [
                {
                    "key": "c",
                    "value": "1"
                }
            ]
        }
    ],
    "range": {
        "start_pos": 0,
        "end_pos": 5,
        "total": 5
    }
}