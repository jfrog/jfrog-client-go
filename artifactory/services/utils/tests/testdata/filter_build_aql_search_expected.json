{
  "results": [
    {
      "repo": "repo1",
      "path": "a",
      "name": "no.in",
      "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
      "actual_sha1": "a",
      "size": 11,
      "created": "2020-06-25T10:21:17.502+03:00",
      "modified": "2020-06-25T10:21:17.510+03:00",
      "properties": [
        {
          "Key": "c",
          "Value": "1"
        },
        {
          "Key": "build.name",
          "Value": "myBuild"
        },
        {
          "Key": "build.number",
          "Value": "1"
        }
      ],
      "type": "file"
    },
    {
      "repo": "repo1",
      "path": "a",
      "name": "yes.in",
      "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
      "actual_sha1": "b",
      "size": 11,
      "created": "2020-06-25T10:21:17.502+03:00",
      "modified": "2020-06-25T10:21:17.510+03:00",
      "properties": [
        {
          "Key": "css",
          "Value": "1"
        },
        {
          "Key": "build.name",
          "Value": "myBuild"
        }
      ],
      "type": "file"
    },
    {
      "repo": "repo1",
      "path": "a",
      "name": "c1.in",
      "actual_md5": "e53098d3d8ee1f5eb38c2ec3c783ef3d",
      "actual_sha1": "c",
      "size": 11,
      "created": "2020-06-25T10:21:17.502+03:00",
      "modified": "2020-06-25T10:21:17.510+03:00",
      "properties": [
        {
          "Key": "c",
          "Value": "1"
        }
      ],
      "type": "file"
    }
  ]
}
